### PR TITLE
Update is-hotkey typings to support React.KeyboardEvent

### PR DESCRIPTION
**Pull request by lynxtaa**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: Fixes https://user-images.githubusercontent.com/22496946/125606192-6e74add5-e2eb-4168-8fa2-ad9e785cc74a.png
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

File: types/is-hotkey/index.d.ts [hunk 1]
```diff
@@ -4,6 +4,10 @@
 //                 Kalley Powell <https://github.com/kalley>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
+import * as React from 'react';
```
Comment (RyanCavanaugh) [inline types/is-hotkey/index.d.ts:7]: The react typings are quite heavy and churn a lot, but this library surface is very small. We also need to consider things like preact or other libraries that might be imported in a way that would create conflict with react. I think this dependency needs to be removed and replaced with an inline structural match for `React.KeyboardEvent`

**Review by RyanCavanaugh — CHANGES_REQUESTED** [review]

File: types/is-hotkey/index.d.ts (hunk 1 above)
Comment (lynxtaa) [reply types/is-hotkey/index.d.ts]: Thanks for the suggestion! I've removed React as dependency

**Comment by peterblazejewicz** [comment]
@lynxtaa mind adding yourself to the owners? I'll merge immediately, as the change looks ok

**Comment by lynxtaa** [comment]
> 
> 
> @lynxtaa mind adding yourself to the owners? I'll merge immediately, as the change looks ok

@peterblazejewicz Done

**Review by peterblazejewicz — APPROVED** [review]
LGTM!
@lynxtaa thx!

**Comment by lynxtaa** [comment]
Ready to merge

**Pull request merged**